src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+import java.util.List;
+
+public class Calculator {
+
+    public double calculate(List<Integer> operands, List<Character> operators) {
+        double result = operands.get(0);
+        int numOfOperations = operators.size();
+        int times = 0;
+
+        while (times < numOfOperations) {
+            result = fourFundamentalArithmeticOperations(result, operators.get(times), operands.get(++times));
+        }
```

Comment:
times라는 변수명:
 애플리케이션의 중요한 로직을 처리하는 부분인데 개인적으로 times라는 변수명은 그 목적이 불분명한 것 같아요.
operatorPosition와 같은 변수명이면 좋을 것 같아요 😃 
마찬가지로 operands 리스트에서 요소를 가져오는 ++times도 rightOperandPosition와 같은 변수명은 어떨까요?

```suggestion
        double result = operands.get(0);
        int numOfOperations = operators.size();
        int operatorPosition = 0;

        while (operatorPosition < numOfOperations) {
            int rightOperandPosition = operatorPosition + 1;
            result = calculateExpression(result, operators.get(operatorPosition), operands.get(rightOperandPosition));
            operatorPosition++;
        }
```